tensorboard/webapp/metrics/views/right_pane/settings_view_component.ng.html

```diff
@@ -52,6 +52,17 @@ <h3 class="section-title">General</h3>
     </div>
   </div>
 
+  <div
+    class="control-row scalars-step-selector"
+    *ngIf="isScalarStepSelectorFeatureEnabled"
+  >
+    <mat-checkbox
+      [checked]="isScalarStepSelectorEnabled"
+      (change)="stepSelectorEnableToggled.emit()"
```

Comment:
btw there will be some conflict with #5790 rename variables
`stepSelectorEnableToggled` --> `stepSelectorToggled`